lib/results.js

```diff
@@ -81,8 +81,9 @@ Results.prototype.createStream = function (opts) {
         self._stream.pipe(output);
     }
 
-    if (!this._isRunning) {
-        this._isRunning = true;
+    if (!self._isRunning) {
+        self._isRunning = true;
```

Comment:
why this change?

Reply:
not sure, left over after my revert. i checked out the file from a previous commit and committed now so should be fixed.